### ISSUE TITLE
YouTube Not Working

### ISSUE THREAD

**Issue by TheRealSonicFan** [halitsever/girici-web-proxy#4]
It seems that videos are not being loaded on YouTube. Is this the expected symptom for the current version?

![alt text](https://i.imgur.com/7vdpzQd.png)

**Comment by halitsever** [comment]
Yeah it seems like youtube is not working, i'm gonna fix it on next PR.
